triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceTest.scala

```diff
@@ -41,12 +41,21 @@ import com.daml.ledger.client.LedgerClient
 import com.daml.ledger.client.services.commands.CompletionStreamElement
 import com.daml.lf.data.Ref.PackageId
 import com.daml.timer.RetryStrategy
+import com.daml.test.evidence.tag.Security.SecurityTest.Property.{
+  Authentication,
+  Authorization,
+  Availability,
+  Confidentiality,
+}
+import com.daml.test.evidence.tag.Security.SecurityTest
+import com.daml.test.evidence.scalatest.ScalaTestSupport.Implicits._
 import com.google.protobuf.empty.Empty
 import com.typesafe.scalalogging.StrictLogging
 import org.scalatest.time.{Seconds, Span}
 
 import java.nio.file.Files
 import scala.concurrent.duration._
+import scala.language.reflectiveCalls
```

Comment:
Please put this locally in the smallest scope that needs it.

Reply:
Done.